proto-lens-discrimination/src/Data/ProtoLens/Group.hs

```diff
@@ -0,0 +1,96 @@
+-- | This module implements discriminators of protobufs generically.
+
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE Rank2Types #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Data.ProtoLens.Group
+    (
+    -- * Comparing Protobuf Messages
+      eqMessage
+
+    -- * 'Group's for Messages
+    , groupingMessage
+    , groupingFields
```

Comment:
I suggest putting `groupingFields` and `groupingField` in a separate Haddock section (or the same one as `groupingFieldValue`.  Unlike `groupingMessage`, I'd expect they're pretty unlikely to be used by real-world programs.

(Alternately, just don't export them.)

Reply:
Moved them to a subsection.